learn/deployment/azurefunctions.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: ballerina-left-nav-pages
+title: Azure Functions
+description: See how the Ballerina deployment in Azure Functions works
+keywords: ballerina, programming language, serverless, cloud, Azure, Functions
+permalink: /learn/deployment/azurefunctions/
+active: azurefunctions
+redirect_from:
+  - /learn/deployment/azurefunctions
+---
+
+# Azure Functions
+
+The Azure Functions extension provides the functionality to expose a Ballerina function as an Azure Function. This is done by importing the `ballerinax/azure.functions` module and simply annotating the Ballerina function with the `functions:Function` annotation. 
```

Comment:
```suggestion
The Azure Functions extension provides the functionality to expose a Ballerina function as an Azure function. This is done by importing the `ballerinax/azure.functions` module and simply annotating the Ballerina function with the `functions:Function` annotation. 
```

Reply:
"Azure Function" is the actual service/entity, that is why it was capitalized.

Reply:
Actually, the product is "Azure Functions" .. I guess we can't say something as a "Azure Function". Let me rename that with something else. I can probably say, "The Ballerina Azure Functions extension provides the functionality to expose a Ballerina function as a serverless function in the Azure Functions platform."

Reply:
+1